webapp/tests/karma/ts/modules/messages/messages-content.component.spec.ts

```diff
@@ -245,5 +246,51 @@ describe('MessagesContentComponent', () => {
       expect(removeMessageFromSelectedConversationSpy.args[0]).to.deep.equal(['message2']);
     });
   });
+
+  describe.only('New message section', () => {
+    it('should display the "new message" section in a conversation with a known recipient', async () => {
+      component.selectedConversation = {
+        'id': '7aeaa4bc-ae5e-4964-8771-697209d2c6dd',
+        'messages': [
+          { 'id': '68e4ae1a-bb6c-4561-98e7-3f7cdfd00486' }
+        ],
+        'contact': {
+          'doc': { 'name': 'Groot' }
+        }
+      };
+
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      const newMessageInput = fixture
+        .debugElement
+        .query(By.css('#message-footer textarea[name="message"]'))
+        ?.nativeElement;
+      expect(newMessageInput).to.not.be.undefined;
```

Comment:
Please add an assertion about the non-existence of `#unknown-contact-error-message`. Right now this only tests that the textarea is displayed. 
If we regressed and showed the unknown error message for known contacts, this test would not fail.